app/src/main/java/io/homeassistant/companion/android/util/vehicle/DomainChecks.kt

```diff
@@ -42,3 +44,7 @@ fun canNavigate(entity: Entity<*>): Boolean {
             ((entity.attributes as? Map<*, *>)?.get("longitude") as? Double != null)
         )
 }
+
+fun alarmHasNoCode(entity: Entity<*>): Boolean {
+    return (entity.attributes as? Map<*, *>)?.get("code_format") as? String == null

```

Comment:
Shouldn't this include a check for the domain, in case there are any other entities with a `code_format` attribute out there?